.github/workflows/update_tgs_dmapi.yml

```diff
@@ -19,12 +19,20 @@ jobs:
         git checkout tgs-dmapi-update
         git reset --hard master
 
+    - name: Install Deps
+      run: sudo apt install dos2unix
+
```

Comment:
Undocumented change?

Reply:
This is what enforces the LF since it force converts it all

Reply:
Fair enough